src/rpc/net.cpp

```diff
@@ -638,7 +636,6 @@ static RPCHelpMan getnetworkinfo()
     if (node.peerman) {
         obj.pushKV("localrelay", !node.peerman->IgnoresIncomingTxs());
     }
-    obj.pushKV("timeoffset",    GetTimeOffset());
```

Comment:
Maybe keep this field and instead return the  median offset of the currently connected outbounds or so? See also https://github.com/bitcoin/bitcoin/pull/24606#issuecomment-1107815177